### PR TITLE
feat: Explicitly expose globals for wrapper

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: src/script/view_model/content/EmojiInputViewModel.js [hunk 1]
```diff
@@ -18,6 +18,7 @@
  */
 
 import * as StorageUtil from 'Util/StorageUtil';
+import {getCursorPixelPosition} from 'Util/PopupUtil.js';
```
Comment (ffflorian) [inline src/script/view_model/content/EmojiInputViewModel.js]:
```suggestion
import {getCursorPixelPosition} from 'Util/PopupUtil';
```

File: src/script/util/wrapper.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    window.z.event = window.z.event || {};
+    window.z.event.WebApp = WebAppEvents;
+
+    window.z.util = window.z.util || {};
+    window.z.util.Environment = Environment;
+
+    window.z.team = window.z.team || {};
+    window.z.team.ROLE = ROLE;
+  }
+}
```
Comment (ffflorian) [inline src/script/util/wrapper.js]: Let's use TS for that

**Comment by ffflorian** [comment]
Did you test this with the wrapper yet?

**Comment by AndyLnd** [comment]
> Did you test this with the wrapper yet?

Yes.

**Pull request merged**